modules/Core/src/main/java/org/terasology/core/logic/door/DoorSystem.java

```diff
@@ -182,18 +183,39 @@ private boolean canAttachTo(Vector3i doorPos, Side side) {
     @ReceiveEvent(components = {DoorComponent.class, BlockRegionComponent.class, LocationComponent.class})
     public void onFrob(ActivateEvent event, EntityRef entity) {
         DoorComponent door = entity.getComponent(DoorComponent.class);
-        Side newSide = (door.isOpen) ? door.closedSide : door.openSide;
+        if (door.isOpen) {
```

Comment:
As mentioned in Slack, you send the CloseDoorEvent in both logic branches, I assume one should be open?